docs/en/get_started.md

```diff
@@ -1,275 +1,220 @@
 
-```python
-dist_params = dict(backend='nccl', port=29500)
+```{note}
+Installing CUDA runtime libraries is enough if you follow our best practices,
+because no CUDA code will be compiled locally. However if you hope to compile
+MMCV from source or develop other CUDA operators, you need to install the
+complete CUDA toolkit from NVIDIA's [website](https://developer.nvidia.com/cuda-downloads),
+and its version should match the CUDA version of PyTorch. i.e., the specified
+version of cudatoolkit in `conda install` command.
 ```
 
-In `config2.py`,
+### Install MMCV without MIM
 
-```python
-dist_params = dict(backend='nccl', port=29501)
-```
+MMCV contains C++ and CUDA extensions, thus depending on PyTorch in a complex
+way. MIM solves such dependencies automatically and makes the installation
+easier. However, it is not a must.
+
+To install MMCV with pip instead of MIM, please follow
+[MMCV installation guides](https://mmcv.readthedocs.io/en/dev-2.x/get_started/installation.html).
+This requires manually specifying a find-url based on PyTorch version and its CUDA version.
 
-Then you can launch two jobs with `config1.py` ang `config2.py`.
+For example, the following command install mmcv-full built for PyTorch 1.10.x and CUDA 11.3.
 
 ```shell
-CUDA_VISIBLE_DEVICES=0,1,2,3 GPUS=4 ./tools/slurm_train.sh ${PARTITION} ${JOB_NAME} config1.py ${WORK_DIR}
-CUDA_VISIBLE_DEVICES=4,5,6,7 GPUS=4 ./tools/slurm_train.sh ${PARTITION} ${JOB_NAME} config2.py ${WORK_DIR}
+pip install "mmcv-full>=2.0rc0" -f https://download.openmmlab.com/mmcv/dist/cu113/torch1.10/index.html
 ```
 
-### Train with IPU
+### Install on CPU-only platforms
 
-The process of training on the IPU is consistent with single GPU training. We just need to have IPU machine and environment
-and add an extra argument `--ipu-replicas ${IPU_NUM}`
+MMClassification can be built for CPU only environment. In CPU mode you can train, test or inference a model.
 
-## Useful tools
+Some functionalities are gone in this mode, usually GPU-compiled ops. But don't
+worry, almost all models in MMClassification don't depends on these ops.
 
-We provide lots of useful tools under `tools/` directory.
+### Install on Google Colab
 
-### Get the FLOPs and params (experimental)
+[Google Colab](https://research.google.com/) usually has PyTorch installed,
+thus we only need to install MMCV and MMClassification with the following
+commands.
 
-We provide a script adapted from [flops-counter.pytorch](https://github.com/sovrasov/flops-counter.pytorch) to compute the FLOPs and params of a given model.
+**Step 1.** Install [MMEngine](https://github.com/open-mmlab/mmengine) and [MMCV](https://github.com/open-mmlab/mmcv) using [MIM](https://github.com/open-mmlab/mim).
 
 ```shell
-python tools/analysis_tools/get_flops.py ${CONFIG_FILE} [--shape ${INPUT_SHAPE}]
+!pip3 install openmim
+!mim install mmengine "mmcv-full>=2.0rc0"
 ```
 
-You will get the result like this.
+**Step 2.** Install MMClassification from the source.
```

Comment:
Install MMClassification from the source. (Install MMClassification from source.)